apps/services/user-profile/src/app/v2/user-profile.service.ts

```diff
@@ -90,7 +90,7 @@ export class UserProfileService {
       this.formatPhoneNumber(userProfile.mobilePhoneNumber)
 
     return await this.sequelize.transaction(async (transaction) => {
-      const commonArgs = [nationalId, transaction] as const
+      const commonArgs = [nationalId, { transaction, maxRetries: 3 }] as const
```

Comment:
Maybe we should name the property `maxRetries` --> `maxTries` to as it is clearer about total tries, rather than total tries = maxRetries + 1. You can think about it.